rfc/rfc004-package-log.md

```diff
@@ -12,8 +12,34 @@ http://peter.bourgon.org/go-kit/#package-log
 
 ## Scope
 
-To be defined.
+### Key / Value Pairs
+*	Log SHALL use key / value pairs for structured data.
+*	Log MAY preserve type safety when expressing logs in a structured format that supports types
+*	Log SHALL provide a way to set a default k/v set per log context
+*	Log MAY provide some pre-canned default keys (level, time, etc) for convenience
+*	Log MAY allow multiple stored logging contexts each with a different set of k/v pairs.
+*	Log MAY allow per log call adhoc k/v pairs (see Logrus as an example)
+
+### Transport
+*	Log SHALL be transport agnostic with pluggable transports.
+*	Log MAY implement io.MultiWriters allowing broadcast of logs over multiple transports.
+*	Log MAY use channels instead
+*	Log MAY provide some built in transports such as syslog and logstash.
+*	Log MAY use encoding.* above the transport level
+*	Log MAY provide configurable back pressure handling strategies in the case of blocked Writers
+
+### Formats
+*	Log SHALL be format agnostic by providing an interface for log formatting.
+*	Log MAY provide some built in formatters such as RFC3164, JSON, etc.
+
+### Levels
+*	Log SHALL provide a set of defined severity levels.
+*	Log SHALL include severity as a k/v pair and allow setting it through the same mechanism as any other k/v pair
```

Comment:
I find these two points contradictory. If severity is just a k/v pair, same as any other, then shouldn't the package be agnostic to which ones are defined?